src/app-layer-modbus.c

```diff
@@ -1384,6 +1396,7 @@ static uint16_t ModbusProbingParser(uint8_t     *input,
     ModbusHeader *header = (ModbusHeader *) input;
 
     /* Modbus header is 7 bytes long */
+    printf("ModbusProbingParser %d\n", input_len);
     if (input_len < sizeof(ModbusHeader))
         return ALPROTO_UNKNOWN;
 
```

Comment:
Doesn't look like these changes are required - debugging info perhaps? Please remove from the PR.